skimage/measure/_regionprops.py

```diff
@@ -397,36 +421,36 @@ def intensity_image(self):
         )
         return self._intensity_image[self.slice] * image
 
-    def _intensity_image_double(self):
-        return self.intensity_image.astype(np.double)
+    def _image_intensity_double(self):
```

Comment:
Since this is an internal function, I guess the renaming wasn't necessary... Fine, though.